backend/controllers/cellMove/cellMoveUtils.ts

```diff
@@ -50,11 +51,17 @@ export const cellAttributes = [
   { text: 'Multiple occupancy', value: 'MO' },
 ]
 
+export const translateCsra = (csraClassificationCode: string): string => {
+  if (csraClassificationCode === 'not entered') return 'not entered'
```

Comment:
Not sure why you are using a hard-coded string for the input? Why not just use `if (!csraClassification)`?

Oh... it's because of the current code that calls it. What a mess!

Reply:
I could change the code where it's used to not pass in a default, and then return 'not entered' here when undefined is passed in?